src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -79,11 +78,11 @@ public boolean start() throws Exception {
             String thisUrl = baseUrl + Util.rawEncode(getId()) + '/';
             listener.getLogger().printf("%s%n%s or %s%n", input.getMessage(),
                     POSTHyperlinkNote.encodeTo(thisUrl + "proceedEmpty", input.getOk()),
-                    POSTHyperlinkNote.encodeTo(thisUrl + "abort", "Abort"));
+                    POSTHyperlinkNote.encodeTo(thisUrl + "abort", Messages.Abort()));
         } else {
             // TODO listener.hyperlink(…) does not work; why?
             // TODO would be even cooler to embed the parameter form right in the build log (hiding it after submission)
-            listener.getLogger().println(HyperlinkNote.encodeTo(baseUrl, "Input requested"));
+            listener.getLogger().println(HyperlinkNote.encodeTo(baseUrl, Messages.input_requested()));
```

Comment:
Not sure it is a good idea to localize build logs. It won't work anyway

Reply:
got it